app/jobs/resolution_proofing_job.rb

```diff
@@ -76,7 +76,7 @@ def perform(
 
     if IdentityConfig.store.idv_socure_shadow_mode_enabled
       SocureShadowModeProofingJob.perform_later(
-        document_capture_session_result_id: document_capture_session.result_id,
+        document_capture_session_result_id: document_capture_session&.result_id,
```

Comment:
This is to handle an edge case where an exception during processing ResolutionProofingJob would be obscured because `document_capture_session` ends up never being initialized and this line raises a `NoMethodError` instead.

Reply:
I went back and forth about this and decided to just keep enqueing the job because:

- Proofers catch exceptions and return them as the `exception` key on their result, so that code does not typically raise errors in actual use (so this is an edge case)
- There is [a code path](https://github.com/18F/identity-idp/blob/main/app/jobs/socure_shadow_mode_proofing_job.rb#L33-L36) in place in the shadow mode job to log an event if something is weird with the document capture session availability